### DISCUSSION TITLE
A small metro track simulation project

### DISCUSSION THREAD

**Discussion by oktay578** [johnrm174/model-railway-signalling#42] [Ideas]
Hello Mr John,

My name is Oktay from Turkey. When I was searching how can I do a small track simulator for my master thesis I found your sources. But I cannot run your project. We have here city flo 350 signalling system that signals have three aspects. If I summurize my goals,  I want to show setting route principle according to interlocks in the a point machine area. Please if you could see this , could you contact with me via ''vuraloktay78@gmail.com'' which is my mail.  You can see under the pape I attached  which is thinking for my projects area could be like this. I am looking forward to an affirmative and inspiring answer from you.

Best wishes / Oktay.

**Comment by johnrm174** [comment]
Hi oktay - I emailed you a sample layout file - did you have any luck with it?

**Comment by oktay578** [comment]
Hey John,

Sorry for the late reply, I have been very stressful and busy times right
now. I received your email but I didn't have any time to look at it. I got
a promotion in my job. So, when I get a chance to look at it, I will let
you know.



Red+white aspect means that we have Route,

Green aspect means that we have the route to next signal.

Red ; Stop

This could be different your signal 🚦 system.

We are using city Flo 350 Bombardier Signaling system here. The system also
provides to possible to set route rail circuit to other rail circuit.


Best regards/ Oktay.

**Comment by oktay578** [comment]
Hi John,

Thanks for getting back to me,  I can not open the simulation program.
Attached indicated failure .

Also, Could you please send me an email address, I can not find our
past emails.

Br, Oktay.
